v2/sprocs/admin_assessment_question_number.sql

```diff
@@ -0,0 +1,29 @@
+DROP FUNCTION IF EXISTS admin_assessment_question_number(integer);
```

Comment:
This function is correct but I think it's a bit more complicated than necessary. Using a [JOIN LATERAL](https://www.postgresql.org/docs/current/static/queries-table-expressions.html#QUERIES-LATERAL) we can also do:

```sql
DROP FUNCTION IF EXISTS admin_assessment_question_number(integer);

CREATE OR REPLACE FUNCTION
    admin_assessment_question_number (
        assessment_question_id integer
    ) RETURNS text
AS $$
SELECT
    CASE
        WHEN questions_in_same_group.count = 1 THEN ag.number::text
        ELSE ag.number::text || '.' || aq.number_in_alternative_group::text
    END
FROM
    assessment_questions AS aq
    JOIN alternative_groups AS ag ON (ag.id = aq.alternative_group_id)
    JOIN LATERAL (SELECT count(*) FROM assessment_questions WHERE alternative_group_id = ag.id) AS questions_in_same_group ON TRUE
WHERE
    aq.id = assessment_question_id;
$$ LANGUAGE SQL;
```